CapacitorRadar.podspec

```diff
@@ -11,6 +11,6 @@
     s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
     s.ios.deployment_target  = '12.0'
     s.dependency 'Capacitor'
-    s.dependency 'RadarSDK', '~> 3.8.11'
+    s.dependency 'RadarSDK', '3.9.2-beta.1'
```

Comment:
Will point at non-beta iOS and Android versions once we release https://github.com/radarlabs/radar-sdk-ios/pull/279 and https://github.com/radarlabs/radar-sdk-android/pull/317